tests/nn/util_test.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=invalid-name, no-self-use,too-many-public-methods
+# pylint: disable=invalid-name,no-self-use,too-many-public-methods,bad-continuation,bad-whitespace
```

Comment:
same here, with the global disable statements. Please remove - if you add these, it means that pylint won't complain when other people add tests to this file, and they might format things considerably worse than you. Once we drop standards once, even for small stuff like this, it becomes 1) difficult to argue that people should maintain the standards we set and 2) easier to write poor code. All you need to do is delete some spaces and align the arrays correctly.